src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -153,3 +211,88 @@
    :post [(are-queries-different? req %)]}
   (restrict-query ["=" "title" title]
                   req))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Conversion/validation of query parameters
+
+(defn parse-boolean
```

Comment:
There's a couple of these kinds of functions here. They're to deal with POST data being JSON and already recognized as a boolean/int etc. I'm thinking once we get the endpoints moved over, we move this coercion to being only needed on the GET side and downstream it assumes it's of the right format.